# Emissions data of the Dow 30

I collected the greenhouse gas (GHG) emissions from the 30 companies in the Dow Jones Industrial Average in order to analyze which companies are emitting the most GHGs. 

I also included emissions data from all world organizations in order to see how the top American emitters compare to other countries' top emitters.

First, here is a brief primer on the different 'scopes' of emissions:
Scope 1: considered direct emissions by a company. These emissions are the result of a company's primary operations. As an example, a delivery company counts emissions from vehicles' exhaust during all deliveries.

Scope 2: considered indirect emissions by a company. These emissions are the result of a company's purchased electricity to operate buildings and operations. As an example, a company's office spaces must count the emissions from the electricity, natural gas or whatever power was generated to operate their facilities.

Scope 3: considered indirect emissions by a company. These emissions are the result of a company's supply chain operations. As an example, a gas & oil refining company must count the emissions used "upstream" in its supply chain, which would be the oil drilling. The company must also count the emissions used "downstream" which would be the exhaust from all end users of the refined oil, such as car drivers.

Also, for measurements, the Dow companies' emissions are reported in metric tons of carbon dioxide equivalents (CO2e). And the world entities' are reported in millions of metric tonnes of CO2e.

In [ ]:
import plotly.express as px
import plotly.graph_objects as go
import dash
from dash import dcc, html
import numpy as np
import pandas as pd

dow = pd.read_excel('dow30_emissions.xlsx', sheet_name='cleaner_data')
world = pd.read_csv('emissions_low_granularity.csv')

In [38]:
dow.info()
dow.head()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 30 entries, 0 to 29
Data columns (total 7 columns):
 #   Column   Non-Null Count  Dtype 
---  ------   --------------  ----- 
 0   company  30 non-null     object
 1   ticker   30 non-null     object
 2   year     30 non-null     int64 
 3   scope1   30 non-null     int64 
 4   scope2   30 non-null     object
 5   scope3   30 non-null     object
 6   offsets  29 non-null     object
dtypes: int64(2), object(5)
memory usage: 1.8+ KB


company ticker  year    scope1   scope2     scope3 offsets
0            Amazon   AMZN  2022  13400000  2890000   54980000      NR
1  American Express    AXP  2022     29422    90583      12025   45296
2             Amgen   AMGN  2022    165000       NR         NR      NR
3             Apple   AAPL  2022     55200     3000   20545800  324100
4            Boeing     BA  2022    642000  1180000  385186000      NR

In [39]:
dow = dow.fillna('NR')
dow.loc[dow['company'] == 'Amgen', 'scope2'] = 0
dow['scope1+2'] = dow['scope1'] + dow['scope2']
dow.info()
dow

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 30 entries, 0 to 29
Data columns (total 8 columns):
 #   Column    Non-Null Count  Dtype 
---  ------    --------------  ----- 
 0   company   30 non-null     object
 1   ticker    30 non-null     object
 2   year      30 non-null     int64 
 3   scope1    30 non-null     int64 
 4   scope2    30 non-null     object
 5   scope3    30 non-null     object
 6   offsets   30 non-null     object
 7   scope1+2  30 non-null     object
dtypes: int64(2), object(6)
memory usage: 2.0+ KB


company ticker  year     scope1      scope2      scope3  \
0                Amazon   AMZN  2022   13400000     2890000    54980000   
1      American Express    AXP  2022      29422       90583       12025   
2                 Amgen   AMGN  2022     165000           0          NR   
3                 Apple   AAPL  2022      55200        3000    20545800   
4                Boeing     BA  2022     642000     1180000   385186000   
5           Caterpillar    CAT  2022     740000     1540000  1226000000   
6                 Cisco   CSCO  2022      34931      672385    17845589   
7               Chevron    CVX  2022  106000000     5000000  2484000000   
8         Goldman Sachs     GS  2022      11980      167991       57233   
9            Home Depot     HD  2022     597000      960000     3696500   
10            Honeywell    HON  2022    1059105     1391346    18201290   
11                  IBM    IBM  2022      78000      577000      513000   
12                Intel   INTC  2022    1190900      347600    22791000   
13    Johnson & Johnson    JNJ  2022     384622      904662     9916211   
14            Coca-Cola     KO  2022    4400000     3500000    57000000   
15       JPMorgan Chase    JPM  2022     102424     1563217      181004   
16           McDonald's    MCD  2021     113286      469236    56803958   
17                   3M    MMM  2022    2420000     2385000    11636000   
18           Merck & Co    MRK  2022    1515000      242000     6680000   
19            Microsoft   MSFT  2022     139413      288029    12571000   
20                 Nike    NKE  2022      50868       24900     9953491   
21     Procter & Gamble     PG  2022    2158000      151000   172597411   
22  Travelers Companies    TRV  2022      17828       20322       10094   
23   Unitedhealth Group    UNH  2022     122356      767792    13788578   
24           Salesforce    CRM  2022       6000      386000      873000   
25              Verizon     VZ  2022     273904     6573720    14401431   
26                 Visa     V   2022       6400       60900      403900   
27              Walmart    WMT  2022    7370000  16170000.0          NR   
28          Walt Disney    DIS  2022     901714     1381568          NR   
29                  Dow    DOW  2022   27290000   7640000.0    80550000   

     offsets    scope1+2  
0         NR    16290000  
1      45296      120005  
2         NR      165000  
3     324100       58200  
4         NR     1822000  
5         NR     2280000  
6         NR      707316  
7   10000000   111000000  
8      79053      179971  
9         NR     1557000  
10        NR     2450451  
11        NR      655000  
12        NR     1538500  
13     27682     1289284  
14        NR     7900000  
15    189327     1665641  
16        NR      582522  
17        NR     4805000  
18        NR     1757000  
19        NR      427442  
20        NR       75768  
21        NR     2309000  
22        NR       38150  
23      7947      890148  
24   1096000      392000  
25        NR     6847624  
26     34456       67300  
27        NR  23540000.0  
28    801077     2283282  
29        NR  34930000.0

In [40]:
dow_hilow = dow.sort_values(by='scope1+2', ascending=False)
dow_lowhi = dow.sort_values(by='scope1+2')
dow_hilow

company ticker  year     scope1      scope2      scope3  \
7               Chevron    CVX  2022  106000000     5000000  2484000000   
29                  Dow    DOW  2022   27290000   7640000.0    80550000   
27              Walmart    WMT  2022    7370000  16170000.0          NR   
0                Amazon   AMZN  2022   13400000     2890000    54980000   
14            Coca-Cola     KO  2022    4400000     3500000    57000000   
25              Verizon     VZ  2022     273904     6573720    14401431   
17                   3M    MMM  2022    2420000     2385000    11636000   
10            Honeywell    HON  2022    1059105     1391346    18201290   
21     Procter & Gamble     PG  2022    2158000      151000   172597411   
28          Walt Disney    DIS  2022     901714     1381568          NR   
5           Caterpillar    CAT  2022     740000     1540000  1226000000   
4                Boeing     BA  2022     642000     1180000   385186000   
18           Merck & Co    MRK  2022    1515000      242000     6680000   
15       JPMorgan Chase    JPM  2022     102424     1563217      181004   
9            Home Depot     HD  2022     597000      960000     3696500   
12                Intel   INTC  2022    1190900      347600    22791000   
13    Johnson & Johnson    JNJ  2022     384622      904662     9916211   
23   Unitedhealth Group    UNH  2022     122356      767792    13788578   
6                 Cisco   CSCO  2022      34931      672385    17845589   
11                  IBM    IBM  2022      78000      577000      513000   
16           McDonald's    MCD  2021     113286      469236    56803958   
19            Microsoft   MSFT  2022     139413      288029    12571000   
24           Salesforce    CRM  2022       6000      386000      873000   
8         Goldman Sachs     GS  2022      11980      167991       57233   
2                 Amgen   AMGN  2022     165000           0          NR   
1      American Express    AXP  2022      29422       90583       12025   
20                 Nike    NKE  2022      50868       24900     9953491   
26                 Visa     V   2022       6400       60900      403900   
3                 Apple   AAPL  2022      55200        3000    20545800   
22  Travelers Companies    TRV  2022      17828       20322       10094   

     offsets    scope1+2  
7   10000000   111000000  
29        NR  34930000.0  
27        NR  23540000.0  
0         NR    16290000  
14        NR     7900000  
25        NR     6847624  
17        NR     4805000  
10        NR     2450451  
21        NR     2309000  
28    801077     2283282  
5         NR     2280000  
4         NR     1822000  
18        NR     1757000  
15    189327     1665641  
9         NR     1557000  
12        NR     1538500  
13     27682     1289284  
23      7947      890148  
6         NR      707316  
11        NR      655000  
16        NR      582522  
19        NR      427442  
24   1096000      392000  
8      79053      179971  
2         NR      165000  
1      45296      120005  
20        NR       75768  
26     34456       67300  
3     324100       58200  
22        NR       38150

# Top Emitters of Scope 1 & 2

In [41]:
# Ensure 'scope1+2' is treated as numeric for color mapping
dow_lowhi['scope1+2'] = pd.to_numeric(dow_lowhi['scope1+2'], errors='coerce')

fig1 = px.bar(dow_lowhi, x='scope1+2', y='company', orientation='h', 
              title='Total Scope 1 & 2 Emissions per company', 
              color='scope1+2', 
              color_continuous_scale='redor', 
              width=800, height=800)
fig1.update_xaxes(title_text='Scope 1 & 2 Emissions')
fig1.update_yaxes(title_text='')
fig1.show()

In [42]:
scope3_reported = dow[dow['scope3'] != 'NR'].sort_values(by='scope3')
scope3_reported

company ticker  year     scope1     scope2      scope3  \
22  Travelers Companies    TRV  2022      17828      20322       10094   
1      American Express    AXP  2022      29422      90583       12025   
8         Goldman Sachs     GS  2022      11980     167991       57233   
15       JPMorgan Chase    JPM  2022     102424    1563217      181004   
26                 Visa     V   2022       6400      60900      403900   
11                  IBM    IBM  2022      78000     577000      513000   
24           Salesforce    CRM  2022       6000     386000      873000   
9            Home Depot     HD  2022     597000     960000     3696500   
18           Merck & Co    MRK  2022    1515000     242000     6680000   
13    Johnson & Johnson    JNJ  2022     384622     904662     9916211   
20                 Nike    NKE  2022      50868      24900     9953491   
17                   3M    MMM  2022    2420000    2385000    11636000   
19            Microsoft   MSFT  2022     139413     288029    12571000   
23   Unitedhealth Group    UNH  2022     122356     767792    13788578   
25              Verizon     VZ  2022     273904    6573720    14401431   
6                 Cisco   CSCO  2022      34931     672385    17845589   
10            Honeywell    HON  2022    1059105    1391346    18201290   
3                 Apple   AAPL  2022      55200       3000    20545800   
12                Intel   INTC  2022    1190900     347600    22791000   
0                Amazon   AMZN  2022   13400000    2890000    54980000   
16           McDonald's    MCD  2021     113286     469236    56803958   
14            Coca-Cola     KO  2022    4400000    3500000    57000000   
29                  Dow    DOW  2022   27290000  7640000.0    80550000   
21     Procter & Gamble     PG  2022    2158000     151000   172597411   
4                Boeing     BA  2022     642000    1180000   385186000   
5           Caterpillar    CAT  2022     740000    1540000  1226000000   
7               Chevron    CVX  2022  106000000    5000000  2484000000   

     offsets    scope1+2  
22        NR       38150  
1      45296      120005  
8      79053      179971  
15    189327     1665641  
26     34456       67300  
11        NR      655000  
24   1096000      392000  
9         NR     1557000  
18        NR     1757000  
13     27682     1289284  
20        NR       75768  
17        NR     4805000  
19        NR      427442  
23      7947      890148  
25        NR     6847624  
6         NR      707316  
10        NR     2450451  
3     324100       58200  
12        NR     1538500  
0         NR    16290000  
16        NR      582522  
14        NR     7900000  
29        NR  34930000.0  
21        NR     2309000  
4         NR     1822000  
5         NR     2280000  
7   10000000   111000000

# Top Emitters of Scope 3

In [43]:
scope3_reported['scope3'] = pd.to_numeric(scope3_reported['scope3'], errors='coerce')

fig2 = px.bar(scope3_reported, x=scope3_reported['scope3'], y=scope3_reported['company'], 
              title='Scope 3 Emissions per company', orientation='h', color='scope3', 
              color_continuous_scale='redor', width=800, height=800)
fig2.update_xaxes(title_text='Scope 3 Emissions')
fig2.update_yaxes(title_text='')
fig2.show()

In [44]:
world.info()
world.head()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 6069 entries, 0 to 6068
Data columns (total 4 columns):
 #   Column                  Non-Null Count  Dtype  
---  ------                  --------------  -----  
 0   year                    6069 non-null   int64  
 1   parent_entity           6069 non-null   object 
 2   parent_type             6069 non-null   object 
 3   total_emissions_MtCO2e  6069 non-null   float64
dtypes: float64(1), int64(1), object(2)
memory usage: 189.8+ KB


year                   parent_entity         parent_type  \
0  1962  Abu Dhabi National Oil Company  State-owned Entity   
1  1963  Abu Dhabi National Oil Company  State-owned Entity   
2  1964  Abu Dhabi National Oil Company  State-owned Entity   
3  1965  Abu Dhabi National Oil Company  State-owned Entity   
4  1966  Abu Dhabi National Oil Company  State-owned Entity   

   total_emissions_MtCO2e  
0                0.498240  
1                1.050222  
2                4.174018  
3                6.193849  
4                7.562050

In [45]:
world.columns = ['year', 'company', 'type', 'emissions']
world.head()

year                         company                type  emissions
0  1962  Abu Dhabi National Oil Company  State-owned Entity   0.498240
1  1963  Abu Dhabi National Oil Company  State-owned Entity   1.050222
2  1964  Abu Dhabi National Oil Company  State-owned Entity   4.174018
3  1965  Abu Dhabi National Oil Company  State-owned Entity   6.193849
4  1966  Abu Dhabi National Oil Company  State-owned Entity   7.562050

In [46]:
world_2022 = world[world['year'] == 2022].sort_values('emissions', ascending=False)
world_2022.info()
world_2022.head()

<class 'pandas.core.frame.DataFrame'>
Index: 115 entries, 1243 to 5133
Data columns (total 4 columns):
 #   Column     Non-Null Count  Dtype  
---  ------     --------------  -----  
 0   year       115 non-null    int64  
 1   company    115 non-null    object 
 2   type       115 non-null    object 
 3   emissions  115 non-null    float64
dtypes: float64(1), int64(1), object(2)
memory usage: 4.5+ KB


year                   company                type     emissions
1243  2022              China (Coal)        Nation State  12290.379570
4866  2022              Saudi Aramco  State-owned Entity   1962.157585
1381  2022                Coal India  State-owned Entity   1407.031017
2438  2022                   Gazprom  State-owned Entity   1254.526477
3197  2022  National Iranian Oil Co.  State-owned Entity   1208.826677

# Top 20 Global Emitters

In [47]:
world_top20 = world_2022.head(20).sort_values(by='emissions')
world_top20['emissions'] = pd.to_numeric(world_top20['emissions'], errors='coerce')

fig3 = px.bar(world_top20, x=world_top20['emissions'], y=world_top20['company'], 
              title='Top 20 World Emitters by Entity Type', orientation='h', color='type',
              color_discrete_sequence=['brown', 'orange', 'red'], width=800, height=800)
fig3.update_xaxes(title='Total Emissions')
fig3.update_yaxes(title='')
fig3.show()

In [48]:
grouped_world = world.groupby('type')['emissions'].sum().round().reset_index()
grouped_world['percentage'] = ((grouped_world['emissions'] / grouped_world['emissions'].sum()) * 100).round(1)
grouped_world.info()
grouped_world

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3 entries, 0 to 2
Data columns (total 3 columns):
 #   Column      Non-Null Count  Dtype  
---  ------      --------------  -----  
 0   type        3 non-null      object 
 1   emissions   3 non-null      float64
 2   percentage  3 non-null      float64
dtypes: float64(2), object(1)
memory usage: 204.0+ bytes


type  emissions  percentage
0  Investor-owned Company   440026.0        31.0
1            Nation State   516196.0        36.3
2      State-owned Entity   464798.0        32.7

# Global Emitters by Entity Type

In [49]:
fig4 = px.bar(grouped_world, x=grouped_world['type'], y=grouped_world['emissions'], 
              title='Total Emissions by Entity Type', color='type',
              color_discrete_sequence=['brown', 'orange', 'red'], width=800, height=800)
fig4.update_layout(showlegend=False)
fig4.update_xaxes(title='Total Emissions')
fig4.update_yaxes(title='')
fig4.show()

In [50]:
djia_photo = 'https://cdn.mos.cms.futurecdn.net/ogy5rmQdTkrx5oxDyYnLQ9.jpg'
world_photo = 'https://www.shutterstock.com/image-vector/concept-reduce-co2-emission-using-260nw-2304197595.jpg'

app = dash.Dash(__name__)
app.layout = html.Div([
    html.Img(src=djia_photo, style={'height': '25%', 'width': '25%', 'display': 'block', 'margin-left': 'auto', 'margin-right': 'auto'}),
    html.H1("The Dow Jones Industrial Average Companies' GHG Emissions"),
    dcc.Graph(id='Dow Companies Scope 1 & 2 emissions', figure=fig1),
    dcc.Graph(id='Dow Companies Scope 3 emissons', figure=fig2),
    html.Img(src=world_photo, style={'height': '50%', 'display': 'block', 'margin-left': 'auto', 'margin-right': 'auto'}),
    html.H1("The Top 20 GHG Emissions Producers in the World"),
    dcc.Graph(id='Top Global GHG Producers', figure=fig3),
    dcc.Graph(id='Emissions by Type', figure=fig4)])
if __name__ == '__main__':
    app.run_server(debug=True)

# Conclusion

Scope 1 & 2: In the first graph, I combined each company's scope 1 and 2 emissions for the reported total. That seemed reasonable since scope 1 is a company's direct emissions, and scope 2 is emissions from energy the company purchases. As we can see, with appoximately 111,000,000 metric tons CO2e (MTCO2e), Chevron produces the greatest amount of emissions by a wide margin. This makes sense because Chevron is an oil and gas company. Chevron's emissions are more than 3 times higher than the second highest emitter in this dataset which is the Dow Company, at almost 35,000.000 MTCO2e. Walmart, Amazon, and Coca-Cola complete the top 5.

Scope 3: Chevron again tops the dataset with almost 2.5 billion scope 3 emissions in 2022. Such high numbers make sense for an oil and gas company since downstream emissions from suppliers, mining, and rigging operations plus upstream emissions from all end users of the oil and gas products must all count in this category. Caterpillar is second in this category with almost 1.3 billion emissions. This also makes sense since the emissions from the buyers of Caterpillar's machines must be counted. Boeing, Proctor & Gamble, and Dow round out the top 5 in scope 3 emissions. 

Top 20 Global Emitters: The top 11 global GHG emitters are either a nation-state, or state-operated entities. Sharewise, nation-states account for over 36% of global emissions, higher than state-owned entities (32.7%) and investor-owned businesses (31%). With over 12 billion MTCO2e emitted in 2022, China Coal, a coal operation run by the government of China is the world's highest GHG emitter. The second highest emitter is the Saudi Arabian, state-owned entity called Saudi Aramco with almost 2 billion MTCO2e. Coal India, Gazprom, and the National Iranian Oil Company (all 3 state-owned entities) round out the top 5 global emitters. The first American company on the list is ExxonMobil in 12th place with over 563 million MTCO2e. 

Global Emitters by Type: With over 516 billion MTCO2e emitted in 2022, nation-states are the top global emitting organizations. State-owned entities are in second with almost 465 billion MTCO2e. And investor-owned companies emitted approximately 440 billion. 

Interesting Side Note: The top GHG emitter in the world is the government of China's coal operations. According to the 2023 Carbon Majors Report, from 2016-2022, China Coal contributed over 25% of the world's emissions. The second highest emitter in that category was Saudi Aramco (state run entity in Saudi Arabia) with 4.8% of global emissions. China also operates cement manufacturing which contributed 3.2% of global emissions. So just China Coal and China Cement emit almost 30% of all global emissions. 

## Resources

I collected the Dow Jones Industrial Average companies' GHG emissions from each company's Sustainability Report, which they provide on their public websites. I gathered each company's data and created a dataset in Microsoft Excel.

The world GHG emitters data came from the following website: https://carbonmajors.org/Downloads

https://carbonmajors.org/briefing/The-Carbon-Majors-Database-26913